ReactNative/ReactTwitter/core/views/debug-screen.js

```diff
@@ -35,6 +39,13 @@ var DebugScreenView = React.createClass({
 
   pushTweetsList () {
     this.props.navigator.push({id: 'tweets-list-identifier'})
+  },
+
+  pushTweetView () {
+    this.props.navigator.push({
+      id: 'tweet-view-identifier',
+      tweetId: '210462857140252672'
```

Comment:
This is how we pass extra params to the navigator route, which can be read from the receiving component accessing its props.